R/ci.R

```diff
@@ -290,3 +296,93 @@ use_azure_coverage_badge <- function() {
   img <- glue("https://img.shields.io/azure-devops/coverage/{github_repo_spec()}/2")
   use_badge("Azure pipelines coverage status", url, img)
 }
+
+#' @section `use_github_actions()`:
+#' Adds a basic ‘r.yaml’ file to the `.github/workflows` directory of a
+#'  package. This is a configuration file for the GitHub Actions service'
```

Comment:
```suggestion
#'  package. This is a configuration file for the GitHub Actions service.
```

Is there a good primary link for us to include re: GitHub Actions? Maybe this, which I see below?

<https://github.com/features/actions>